demisto_sdk/commands/common/errors.py

```diff
@@ -154,6 +155,7 @@
     "description_contains_contrib_details": {'code': "DS105", 'ui_applicable': False,
                                              'related_field': 'detaileddescription'},
     "invalid_description_name": {'code': "DS106", 'ui_applicable': False, 'related_field': ''},
+    "description_contains_demisto_word": {'code': "DS107", 'ui_applicable': False, 'related_field': ''},
```

Comment:
```suggestion
    "description_contains_demisto_word": {'code': "DS107", 'ui_applicable': True, 'related_field': 'detaileddescription'},
```
You can fix this through the UI by taking the word out of the description